src/Accounts/Accounts/AzureRmAlias/Mappings.json

```diff
@@ -1279,6 +1279,13 @@
     "Set-AzApplicationGatewayWebApplicationFirewallConfiguration": "Set-AzureRmApplicationGatewayWebApplicationFirewallConfiguration",
     "New-AzApplicationGatewayFirewallDisabledRuleGroupConfig": "New-AzureRmApplicationGatewayFirewallDisabledRuleGroupConfig",
     "New-AzApplicationGatewayFirewallExclusionConfig": "New-AzureRmApplicationGatewayFirewallExclusionConfig",
+    "New-AzApplicationGatewayFirewallCondition": "New-AzureRmApplicationGatewayFirewallCondition",
```

Comment:
Please remove all changes to this file - we are only supporting aliases for cmdlets that existed before Az 1.0.0.

Reply:
will change